platform_plugin_forum_email_notifier/email.py

```diff
@@ -27,3 +33,17 @@ def send_forum_email_notification(recipient, language, user_context):
         site=Site.objects.get_current(), user=user_context.get("user")
     ):
         ace.send(msg)
+
+
+def send_digest_email_notification(recipient, language, user_context):
+    """Send email notification for forum events to suscribers."""
+    forum_email_message_type = DigestEmailNotification()
```

Comment:
what's the difference between send_forum_email_notification and this one?

Reply:
The message is the only one. It can be refactored